ConsistencyManager/ConsistencyManager.swift

```diff
@@ -580,25 +597,33 @@ public class ConsistencyManager {
      This function uses the map functionality of the models to generate a new model given a list of modelUpdates.
```

Comment:
Here, and elsewhere throughout the changes in this RB, it would be good to mention projections since at this point it just says "given a list of modelUpdates."

Or maybe it would suffice to just give docs about this in one place in this file.